eng/restore/binplacePackages/binplacePackages.depproj

```diff
@@ -33,6 +33,10 @@
       <Version>4.5.3</Version>
     </PackageReference>
 
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Condition="('$(TargetsNetStandard)' == 'true' AND '$(NETStandardVersion)' &gt;= 2.0) OR '$(TargetsNetStandard)' != 'true'">
```

Comment:
TargetsNetStandard != true OR NS >= 2.0

Reply:
this doesn't seem right. AFAIK for sourcebuild we can't use pre-built bits on netcoreapp. cc @ericstj